docs/sphinx/specification/cudaq/types.rst

```diff
@@ -448,4 +503,4 @@ start in the :code:`|0>` computational basis state.
     }
  
     // std::array-like semantics
-    cudaq::qreg<5> fiveCompileTimeQubits;
\ No newline at end of file
+    cudaq::qreg<5> fiveCompileTimeQubits;
```

Comment:
looks like a qreg got left around, should update to qarray

Reply:
This is on the `qreg` part of the documentation it is marked as deprecated.

Reply:
Ah... my bad 👍